doc/rst/tools/mason/mason.rst

```diff
@@ -287,6 +272,68 @@ Once your package is uploaded, maintain the integrity of your package, and pleas
 chapel team if your package should be taken down. 
 
 
+Local Registries
+================
+
+It is sometimes desirable to use a local registry, for example with libraries
+you don't intend to distribute. The following steps create a local registry
+starting with Bricks for ``ProjectA`` and ``ProjectB`` which were created with
+``mason new ProjectA`` and ``mason new ProjectB``, and are located at
+``/path/to/my/projects/Project[AB]``. It is expected that mason will be
+extended to simplify and handle more of this process.
+
+First create commit, and tag the projects that will be in the registry:
+
+.. code-block:: sh
+
+   cd /path/to/my/projects
```

Comment:
Adding some concise comments on their own line like `# Create project` might help break this up a little and improve readability.